### DISCUSSION TITLE
Performance on 7z compression

### DISCUSSION THREAD

**Discussion by pkellner** [aonez/Keka#1373] [Q&A]
Hi, I bought Keka today because I notice I can set the compression level with 7z's. I have lots video project I create with PR Pro, and every bit of extra compression helps. I've been using cocotech pathfinder and they give no options for compression, and also do not have a way to test 7z integrity without expanding the files. I notice Keka does let me pick a compression option.

So, I picked the highest 7z compression. My results were a little suprising.  8.4Gig compressed with Keka, in about 5 minutes to 5.6Gig.  With Pathfinder, it took about 30 minutes, but compressed to 4.9Gig.  Is that expected? I figured Keka, who specializes in compress would be better.

**Comment by aonez** [comment]
Thanks for the feedback @pkellner. Just did a test with a test video available here:
https://devstreaming-cdn.apple.com/videos/wwdc/2023/10037/4/77026DAB-A5CB-4FD8-8516-5E63DD12C236/downloads/wwdc2023-10037_hd.mp4?dl=1

In my tests:


| File | Size in Bytes | Ratio |
| - | - | - |
| Uncompressed  | 64636706  | 100% |
| Pathfinder (7Z)  | 63084971  | 97.59% |
| Keka (7Z + Slowest)  | 62536441  | 96.75% |

I don't know what configuration Pathfinder uses, but as you can see results can be different with different source files.

Can you try if any of these build results in better compression with your files?

https://github.com/aonez/Keka/releases/download/dev-test-builds/Keka-QS.7z
https://github.com/aonez/Keka/releases/download/dev-test-builds/Keka-QS+LZMA.7z

**Reply by pkellner** [reply]
Keka-QS.7z went to 5.6GB
Keka-QS+LZMA.7z went to 5.6GB

Now, when I compress with PathFinder I get 5.8GB (not sure how this could change, I'm pretty sure I did not get hit in the head between these two tests)
